src/protobuf-net.Core/Meta/TypeModel.cs

```diff
@@ -132,15 +132,17 @@ internal static WireType GetWireType(TypeModel model, DataFormat format, Type ty
                     return WireType.String;
             }
             return WireType.None;
-        }
-
        /// <summary>
        /// Indicates whether a type is known to the model
-        /// </summary>
-        internal virtual bool IsKnownType<T>(CompatibilityLevel ambient)
-            => (TypeHelper<T>.IsReferenceType | !TypeHelper<T>.CanBeNull) // don't claim T?
-            && GetSerializerCore<T>(ambient) is object;
-
-        internal const SerializerFeatures FromAux = (SerializerFeatures)(1 << 30);
-
+        }
+
+        /// <summary>
+        /// Indicates whether a type is known to the model
```

Comment:
the change here is fine as far as it goes, but if you look at the commit delta including whitespace: it's a doozy; honestly, it might be worth just reverting this one file and not worrying about this tweak